ordering/ordering.proto

```diff
@@ -5,8 +5,9 @@ option go_package = "github.com/relab/gorums/ordering";
 
 service Gorums { rpc NodeStream(stream Message) returns (stream Message); }
 
-message Message {
-  uint64 ID = 1;
+message Message {}
```

Comment:
I think I might remove this "placeholder" message and use `Metadata` instead for the `NodeStream`, even though the type that is actually being sent is `gorumsMessage` which contains `Metadata`.

Reply:
Right, I was wondering (but didn't check carefully); do we still need the exported `ordering` package? I see several places `ordering.Message` gets replaced with `gorumsMessage`. I guess this comes back to the question, can we move `NodeStream` and the `Metadata` message into `dev`? Maybe not.

Reply:
We still need the `Metadata` message, unless we want to marshal that manually as well. That means that we can't bundle it, as that will lead to registration conflicts if multiple gorums services are imported in the same project.